client/blindai_preview/_dcap_attestation.py

```diff
@@ -116,6 +116,9 @@ def validate_attestation(
         __package__, "Intel_SGX_Provisioning_Certification_RootCA.pem"
     )
 
+    if len(collateral) < 1:
```

Comment:
It is a bit strange.
We might want to check that the collateral structure from the server match the one that is expected. But as it is it is only a partial check. What if the server removes a field ? 
In short, we might want to check, but if we check we need to do it fully.

Reply:
This is a hack, but the client will simply fail if the collateral is empty, and will not return a proper exception. I had this issue when testing the simulation mode, as the simulation mode will send empty collateral...

Reply:
I wanted at first to check if the exact amount of data are available, but to keep it simple, I am now simply checking this. Anyway, if one field is missing, the client will just crash, atm.

Reply:
I see. We need to change that, but I think we can do it in another PR/commit.